tests/gold_tests/cache/replay/negative-revalidating-enabled.replay.yaml

```diff
@@ -115,10 +115,9 @@ sessions:
         - [ Host, example.com ]
         - [ uuid, 14 ]
 
-          # After this delay, the item is 8 seconds old. This makes it:
-          #  6 seconds beyond the server's max-age of 2 seconds and
-          #  2 seconds beyond ATS's max_stale_age of 6 seconds.
-      delay: 4s
+          # After this delay, the item is 10 seconds old. This is 2 seconds beyond
```

Comment:
Wonder if we should keep the 8sec one (or make it 7sec) to show that its greater than max_stale_age yet still successfully serves stale ... or maybe just change the first stale check to be at 7s?

Reply:
This is checking current age exceeds `max-age + max_stale_age` (8 seconds) and ATS returns 503.

> to show that its greater than max_stale_age yet still successfully serves stale

This case is covered by `proxy_serve_stale` test. 
https://github.com/apache/trafficserver/pull/12241/files#diff-40842de8789380a64ac754078299609752920b52aae3d85a201d1ceab191d650R122-R124